### ISSUE TITLE
Add a project 

### ISSUE THREAD

**Issue by adityasingh-0803** [Niketkumardheeryan/ML-CaPsule#1197]
Add a project on regression with insurance dataset

**Comment by adityasingh-0803** [comment]
assign me under swoc @Niketkumardheeryan

**Comment by Vaibhav2154** [comment]
@adityasingh-0803 I've assigned you the issue.
Reach out if you need any help.
Add your project with proper documentation

**Comment by adityasingh-0803** [comment]
thanks @Vaibhav2154 i want to ask that i will add pynb notebook along with image of quote from which the words will be extracted

**Comment by adityasingh-0803** [comment]
can you explain proper documentation please

**Comment by adityasingh-0803** [comment]
@Niketkumardheeryan ive done the task please check

**Comment by adityasingh-0803** [comment]
@Vaibhav2154

**Comment by adityasingh-0803** [comment]
please tell

**Comment by kashifalikhan36** [comment]
I'm interested, please assign me this issue

**Comment by kashifalikhan36** [comment]
@Vaibhav2154 please assign me as well

**Comment by GitHubPro18** [comment]
@Vaibhav2154 I would like to work on the Detection of Heart Disease using ML but using a different approach. I would like to use the ECG signals data, apply CNN-LSTM based models to capture the spatial as well as the temporal dependencies of the data to identify arrythmia and its types and deploy the project using Streamlit. I'm open to using other frameworks as well for deployment or any other models that may produce a higher accuracy.

**Comment by guptaanshika8168-oss** [comment]
Hii, I would like to work on this issue under GSSoC 2026. Please assign it to me

**Comment by Sid-2709** [comment]
Hi @Vaibhav2154 , I would like to work on this issue under SWOC.

My approach will be:

understand the existing project structure and dataset format
create a regression project using the insurance dataset with proper preprocessing and model training
add evaluation metrics and clear visualizations for better understanding
update the documentation and test everything locally before creating the PR

**Comment by ffrazi** [comment]
Hi @niketkumardheeryan, I am a GSSoC '26 contributor.
I would love to contribute a new project to this repository!Given my background in Computer Vision and NLP, I propose adding a Real-Time Emotion Recognition system or a Retrieval-Augmented Generation (RAG) pipeline for legal document analysis. I have previously:
* Achieved 92% accuracy on emotion classification using TensorFlow and OpenCV.
* Developed a RAG-based AI assistant using LangChain and VectorDB.  I can provide clean, documented code along with a detailed README and sample datasets. Please let me know if I can proceed with one of these!

**Comment by 24pa1a42a9-cpu** [comment]
Hi, I would like to work on this issue under GSSoC'26.
Please assign it to me.

**Comment by 24pa1a42a9-cpu** [comment]
Hi,
I am interested in contributing to this project and would be happy to work on ML/Data Science related tasks and projects.
 I have experience working on AI/ML projects using Python, Scikit-learn, Pandas, and Streamlit. Recently, I built and deployed an AI-powered Customer Churn Prediction web application that predicts whether a customer is likely to churn based on behavioral data.

The model achieved good performance with strong accuracy after data preprocessing, feature engineering, and model evaluation. I also worked on data cleaning, visualization, and deployment of the application.
